README.rst

```diff
@@ -503,6 +503,16 @@ configuration status of a device changes, this ensures the check reacts
 quickly to events happening in the network and informs the user promptly
 if there's anything that is not working as intended.
 
+SNMP DeviceMonitoring
+~~~~~~~~~~~~~~~~~~~~~
+
+This check provides an alternative protocol to collect monitoring information from devices that don't have
+the ability to use `openwisp-config agent <https://github.com/openwisp/openwisp-config/>`_.
+The information is collected via SNMP using `netengine <https://github.com/openwisp/netengine/>`_. The devices
+need to have an SNMP daemon in order for this to work.
+This check is disabled by default, you may choose to enable auto creation of this check by setting
+`OPENWISP_MONITORING_AUTO_SNMP_DEVICEMONITORING <#OPENWISP_MONITORING_AUTO_SNMP_DEVICEMONITORING>`_ to ``True``.
```

Comment:
We should explain here how to use this check (how to enable it).

Can you make the check inline in DeviceAdmin editable so it can be easier to add this check?

Please also add mention of this feature (SNMP support) at the bottom of the feature list in the beginnign of the README, linking the bullet point to this section, which should contain instructions on how to add the check, possibily using the device admin.